app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -147,12 +151,29 @@
   align-self: flex-end;
   flex-shrink: 0;
   width: 100%;
-  border-top: 2px solid var(--box-darker);
+  line-height: 100%;
+  height: 80px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
 
   .selection-status-content {
     display: block;
-    padding: $su-4;
     text-align: center;
+
+    p {
+      font-size: $su-4;
+      font-weight: $fw-bold;
+      line-height: 150%;
```

Comment:
Since you're using Crayons variables for the other properties, you could also use one of the Crayons variables for line-height. To render a 150% line-height, you can say: `line-height: $lh-base;`

Reply:
ah I missed this one. Thanks!